packages/phishing-controller/src/PhishingController.test.ts

```diff
@@ -2587,4 +2582,209 @@ describe('PhishingController', () => {
       expect(scope.isDone()).toBe(true);
     });
   });
+
+  describe('hotlistLastSuccessTimestamp handling', () => {
+    it('should update hotlistLastSuccessTimestamp only on successful response', async () => {
+      // Setup initial state
+      const controller = getPhishingController({
+        state: {
+          phishingLists: [
+            {
+              allowlist: [],
+              blocklist: ['185.149.120.83', '193.3.19.52', '209.159.154.156'],
+              c2DomainBlocklist: [],
+              fuzzylist: [],
+              tolerance: 0,
+              lastUpdated: 1742310773, // Just before the update
+              name: ListNames.MetaMask,
+              version: 0,
+            },
+          ],
+          hotlistLastSuccessTimestamp: 1742310773, // Previous successful timestamp
+        },
+      });
+
+      // Mock a successful response with lastFetchedAt at the root level similar to the example
+      nock(PHISHING_CONFIG_BASE_URL)
```

Comment:
With `nock` we also want to ensure that the request was actually done and the mock was hit. You can do that by using the isDone (https://github.com/nock/nock?tab=readme-ov-file#isdone) expecation from nock:
```js
const scope = nock(...);
...
expect(scope.isDone()).toBe(true);
```